test/src/unittest/io/test_audioloader_streaming.py

```diff
@@ -240,6 +247,40 @@ def testMD5(self):
         self.assertEqual(md5_mp3, "1e5a598218e9b19cfe04d6c2f61f84a6")
         self.assertEqual(md5_ogg, "a87dad40fea0966cc5b967d5412e8868")
         self.assertEqual(md5_aac, "9a4c7f0da68d4b58767f219c48014f9c")
+    
+    def testMultiStream(self):
+        
+        # A multistream file containing the same audio on streams 0 and 1 
+        filename = 'multistream.mka'
```

Comment:
I would reconsider this test - it would also succeed if the audioStream parameter was ignored and both AudioLoaders opened the first stream. Is it possible to have two multistream files, where one has streams a + b and the other b + a?